.github/scripts/tryrebase.py

```diff
@@ -36,6 +39,38 @@ def rebase_onto(pr: GitHubPR, repo: GitRepo, dry_run: bool = False) -> None:
                         "git pull --rebase`)", dry_run=dry_run)
 
 
+def rebase_ghstack_onto(pr: GitHubPR, repo: GitRepo, dry_run: bool = False) -> None:
+    subprocess.run([sys.executable, "-m", "pip", "install", "ghstack"])
```

Comment:
We should probably make this so that it only installs ghstack if it can't run `ghstack --help`, just so that we can still run this locally without it installing ghstack every time